src/main/java/org/osiam/resources/exceptions/OsiamExceptionHandler.java

```diff
@@ -36,10 +35,11 @@
 
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
+
 @ControllerAdvice
 public class OsiamExceptionHandler extends ResponseEntityExceptionHandler {
 
-    private static final Logger LOGGER = Logger.getLogger(OsiamExceptionHandler.class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger(OsiamExceptionHandler.class);
```

Comment:
Ok, that's a good point. I'm looking forward to merge the pull request in auth-server.